static/src/javascripts/projects/common/modules/commercial/liveblog-dynamic-adverts.js

```diff
@@ -61,7 +61,7 @@ define([
 
     function insertAds(slots) {
         for (var i = 0; i < slots.length && slotCounter < MAX_ADS; i++) {
-            var $adSlot = bonzo(createAdSlot('inline1' + slotCounter++, 'liveblog-inline'));
+            var $adSlot = bonzo(createAdSlot('inline1' + slotCounter++, 'liveblog-inline block'));
```

Comment:
It's a bit funny to see both `inline` and `block` side by side. Should we renamed `liveblog-inline` as `liveblog-mpu` or something instead?